schema/go.mod

```diff
@@ -1,7 +1,7 @@
 module cosmossdk.io/schema
 
-// NOTE: this go.mod should have zero dependencies and remain on go 1.12 to stay compatible
+// NOTE: this go.mod should have zero dependencies and remain on go 1.22.2 to stay compatible
 // with all known production releases of the Cosmos SDK. This is to ensure that all historical
 // apps could be patched to support indexing if desired.
 
-go 1.12
```

Comment:
i think this is set as 1.12 for a reason that is stated in the comment above. @tac0turtle can you confirm ?

Reply:
thank you for feedback, if have a conflict i will rollback this state

Reply:
@aaronc did you put it to 1.12 for a reason?

Reply:
Yes. That's the earliest go version cosmos-sdk and gaia were released with in case any one wants to go all the way back to genesis

Reply:
There is literally a comment in the go.mod that explains this that you can see in the PR diff above so I am not sure what the intention of this PR is

Reply:
yeah, thank you, i'm wrong